backend/inspirehep/disambiguation/tasks.py

```diff
@@ -64,3 +68,36 @@ def disambiguate_signatures(self, clusters):
             disambiguation_assigned_clusters.labels("2+").inc()
             LOGGER.debug("Received cluster with more than 1 author.")
     db.session.commit()
+
+
+@shared_task(ignore_result=False, bind=True)
+def disambiguate_authors(self, record_uuid):
+    record = InspireRecord.get_record(record_uuid)
+    if "Literature" not in record["_collections"]:
+        return
+    authors = list(record.get_modified_authors())
```

Comment:
The `list` is not needed, as the `for` loop can iterate on any iterable, which includes generators. It might be needed if you want to reuse it later (because iterating on a generator consumes it), but that doesn't seem to be the case here.
```suggestion
    authors = record.get_modified_authors()
```